graylog2-server/src/main/java/org/graylog/plugins/views/search/views/ViewSummaryService.java

```diff
@@ -40,16 +38,22 @@ protected ViewSummaryService(MongoConnection mongoConnection,
     }
 
     private PaginatedList<ViewSummaryDTO> searchPaginated(DBQuery.Query query,
```

Comment:
searchPaginated() seems to be no longer in use anywhere (according to IntelliJ) - why does ist still exist?